frontend/src/components/ErrorState/ErrorState.tsx

```diff
@@ -99,13 +99,12 @@ export const ErrorState = ({
 									? {
 											to: {
 												pathname: '/?sign_up=1',
-												state: {
-													previousPathName:
-														window.location
-															.pathname,
-												},
 											},
 											href: undefined,
+											state: {
```

Comment:
error `show all sessions for the user` does not work

<img width="543" alt="image" src="https://user-images.githubusercontent.com/1351531/218532546-5ccf7c96-3ccd-438b-b154-94416635affd.png">

Reply:
This is broken on master as well, though I've noticed that state is not passed to the underlying Link of ButtonLink. Fixed the latter in [917f6d6](https://github.com/highlight/highlight/pull/4203/commits/917f6d6329ef39ebdf3e99f25ae5c7bf7be15443)